contracts/hack/SuperFollowModule.sol

```diff
@@ -62,8 +69,11 @@ contract HarbergerTaxStuff {
     address public admin;
 
     // profileId => max number of super followers
+    mapping(uint256 => uint256) maxNumberOfSuperFollowers; // need to set in constructor!!
```

Comment:
Set in the `initializeFollowModule` function @moose-code 
Seems you changed this variable name though...

Reply:
I'll fix it